xbmc/addons/GUIWindowAddonBrowser.cpp

```diff
@@ -184,14 +184,18 @@ bool CGUIWindowAddonBrowser::OnClick(int iItem, const std::string &player)
       g_mediaManager.GetNetworkLocations(shares);
       std::string path;
       if (CGUIDialogFileBrowser::ShowAndGetFile(shares, "*.zip", g_localizeStrings.Get(24041), path))
+      {
         CAddonInstaller::GetInstance().InstallFromZip(path);
+        Refresh();
```

Comment:
I recommend making this a synchronous user interaction. Currently a user don't get proper confirmation on whether the install succeeded or not. It happened several times to me that I missed the short kai toast indicating a failure.